src/app/app.ts

```diff
@@ -81,6 +82,9 @@ class AppImpl implements App {
     createZDSubscription = async (): Promise<AppCallResponse> => {
         // get zendesk client for user
         const zdClient = await newZDClient(this.zdOptions);
+        const req = zdClient.triggers.definitions() || '';
+        const definitions = await tryPromiseWithMessage(req, 'Failed to fetch trigger definitions');
```

Comment:
Here, `req` may be assigned to an empty string, so we are potentially calling `tryPromiseWithMessage` with a string instead of a promise. Is this intended?